integration-tests/cucumber-tests-platform-publiclighting/src/test/resources/features/publiclighting/osgp-adapter-ws-core/Firmwaremanagement/PendingFirmwareUpdate.feature

```diff
@@ -36,13 +36,19 @@ Feature: FirmwareManagement pending firmware update
       | OrganizationIdentification | TestOrganization   |
     When the device sends a register device request to the platform over "<Protocol>"
       | DeviceIdentification | TEST1024010101010 |
-      | Protocol             | <Protocol>        |
       | DeviceUid            | eHW0eEFzN0R2Okd5  |
       | IpAddress            | 127.0.0.2         |
       | DeviceType           | SSLD              |
       | HasSchedule          | false             |
     Then the register device response contains
       | Status | OK |
+    When the device sends a confirm register device request to the platform over "<Protocol>"
+      | DeviceIdentification | TEST1024010101010 |
+      | DeviceUid            | eHW0eEFzN0R2Okd5  |
+      | IpAddress            | 127.0.0.2         |
+      | DeviceType           | SSLD              |
```

Comment:
The step code as defined `org.opensmartgridplatform.cucumber.platform.publiclighting.glue.steps.mocks.OslpDeviceSteps.theDeviceSendsAConfirmRegisterDeviceRequestToThePlatform(String, Map<String, String>)` only uses the `DeviceIdentification` and `DeviceUid`. Therefore, I think `IpAddress` and `DeviceType` can be removed from the data table.

Reply:
Agree

Reply:
I removed the entries from the data table. The feature runs successful and all scenario's/examples pass.